### ISSUE TITLE
Xpath got cut when trying to make screenshot of the webelement. 

### ISSUE THREAD

**Issue by OlgaKozlova** [yandex-qatools/ashot#36]
http://prntscr.com/75f7x7
Here is screenshot; 
May be it is my fault, but all my elements selenium successfully finds in other tests, only trying to make the shot of the element xpath got cut;

**Comment by pazone** [comment]
Hello. Can you show the code snippet when you are taking the screenshot?

**Comment by OlgaKozlova** [comment]
Pavel, I can’t delete my issue on GitHub, but it seems that the problem was with JQuery, not хPath. I tried to use method dealing with JQuery on the page without JQuery.

Problem got solved when used methods of aShot framework not dealing with JQuery instead (new AShot()
        .coordsProvider(new WebDriverCoordsProvider()) //find coordinates with WebDriver API
        .takeScreenshot(driver, element);
)

But xPath really got cut… I noticed it when was trying to debug that issue.

I’m really sorry for disturbing you!

Thanks!

Olga Kozlova
ISsoft, a subsidiary of Coherent Solutions, Inc.
5 Chapaeva str., Minsk, Belarus, 220034
Mobile +375.25.980.94.84
Skype: csi.olgakozlova
www.coherentsolutions.com<http://www.coherentsolutions.com/>
Better Teams. Better Results
